utils/conversion/conversion_wrapper.go

```diff
@@ -0,0 +1,49 @@
+package conversion
+
+import (
+	"fmt"
+	"time"
+
+	specqbft "github.com/ssvlabs/ssv-spec/qbft"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+
+	"github.com/ssvlabs/ssv/exporter/convert"
+)
+
+var (
+	ErrNegativeTime = fmt.Errorf("time can't be negative")
+)
+
+// LenUint64 - Returns the length of the slice as uint64, due length can't be negative
+func LenUint64[T any](slice []T) uint64 {
+	return uint64(len(slice))
+}
```

Comment:
are we sure the linter warns for `uint64(len(slice))`? this would be weird

if not, then this func can be removed imo

Reply:
seems like linter is not warning, otherwise a `#nosec G115` would've been required here.